data/provider/src/main/java/tech/pegasys/teku/api/ChainDataProvider.java

```diff
@@ -675,6 +676,18 @@ protected SyncCommitteeRewardData calculateRewards(
     return data;
   }
 
+  @VisibleForTesting
+  protected int calculateSyncAggregateBlockRewards(UInt64 proposerReward, SyncAggregate aggregate) {
```

Comment:
Would be clearer if this method was named `calculateProposerSyncAggregateBlockRewards`?